Columbus/frontend/src/pages/Profile/Profile.js

```diff
@@ -85,23 +86,23 @@ function Profile({...props}) {
                                                   "biography": ""
                                               });
 
-  //const [sharedPosts, setSharedPosts] = useState(dummyPosts.slice(0,1));
-  const [likedPosts, setLikedPosts] = useState(dummyPosts.slice(1,4));
+  const [isBlocked, setIsBlocked] = useState(false);
   
   const [tabValue, setTabValue] = useState('shared');
   const [infoLoading, setInfoLoading] = useState(true);
 
   const [followingsOpen, setFollowingsOpen] = useState(false);
   const [followersOpen, setFollowersOpen] = useState(false);
   const [editProfileOpen, setEditProfileOpen] = useState(false);
+  const [verticalMenuOpen, setVerticalMenuOpen] = useState(false);
 
 
   useEffect(() => {
 
      USER_SERVICE.GET_PROFILEINFO(userId)
       .then((res) => {
         const proInfo = res.data.response;
-        console.log('profile info ',  res.data.response['username'])
+        //console.log('profile info ',  res.data.response['username'])
```

Comment:
Burada bir console.log kalmış commentli, bunun dışında eline sağlık güzel duruyor.

Reply:
I am handling this, thank you !